src/test/app/CFToken_test.cpp

```diff
@@ -0,0 +1,89 @@
+//------------------------------------------------------------------------------
+/*
+  This file is part of rippled: https://github.com/ripple/rippled
+  Copyright (c) 2023 Ripple Labs Inc.
+
+  Permission to use, copy, modify, and/or distribute this software for any
+  purpose  with  or without fee is hereby granted, provided that the above
+  copyright notice and this permission notice appear in all copies.
+
+  THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+  WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+  MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+  ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+  WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+  ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <ripple/protocol/Feature.h>
+#include <ripple/protocol/jss.h>
+#include <test/jtx.h>
+
+namespace ripple {
+
+class CFToken_test : public beast::unit_test::suite
+{
+    // TODO move into lib?
+    // Helper function that returns the owner count of an account root.
+    static std::uint32_t
+    ownerCount(test::jtx::Env const& env, test::jtx::Account const& acct)
+    {
+        std::uint32_t ret{0};
+        if (auto const sleAcct = env.le(acct))
+            ret = sleAcct->at(sfOwnerCount);
+        return ret;
+    }
+
+    void
+    testEnabled(FeatureBitset features)
+    {
+        testcase("Enabled");
+
+        using namespace test::jtx;
+        {
+            // If the CFT amendment is not enabled, you should not be able to
+            // create CFTokenIssuances
+            Env env{*this, features - featureCFTokensV1};
+            Account const& master = env.master;
+
+            BEAST_EXPECT(ownerCount(env, master) == 0);
+
+            // TODO why not working with short codes?
```

Comment:
@sappenin and @scottschurr and i discussed - the effort to make this work as desired is:

1) actually quite big - requires a whole new field
2) probably not a good idea because it means that if a user inputs a hex-code currency that _DOES_ work in ISO format, when they retrieve it using rpcs like `tx` the transaction will be different (the currency will be in ISO). this means a tool that validates old transactions would break